wa/workloads/meabo/__init__.py

```diff
@@ -72,17 +72,19 @@ class Meabo(Workload):
             default=1048576,
         ),
         Parameter(
-            'nrow',
+            'num_row',
```

Comment:
I would pluralize this to `num_rows`; and similar blow.

Reply:
Updated